Client.xcodeproj/project.pbxproj

```diff
@@ -6693,6 +6698,7 @@
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ALWAYS_SEARCH_USER_PATHS = NO;
 				APPLICATION_EXTENSION_API_ONLY = NO;
+				APP_IDENTIFIER_PREFIX = 9G8J6YA743;
```

Comment:
These are added as `User-defined variables` which are also injected into the Info.plist so we can reference them through the code.

![screen shot 2016-11-08 at 10 03 38 am](https://cloud.githubusercontent.com/assets/434322/20104103/afc716a4-a59a-11e6-9092-ed30a1a6d6f0.png)

![screen shot 2016-11-08 at 10 03 51 am](https://cloud.githubusercontent.com/assets/434322/20104108/b2f5b506-a59a-11e6-9358-f14568700347.png)